target/linux/ramips/dts/WL-WN575A3.dts

```diff
@@ -0,0 +1,128 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
```

Comment:
Please include <[dt-bindings/input/input.h](http://lxr.free-electrons.com/source/include/dt-bindings/input/linux-event-codes.h?v=4.4)> and  <[dt-bindings/gpio/gpio.h](http://lxr.free-electrons.com/source/include/dt-bindings/gpio/gpio.h?v=4.4)> here as well.

Use the constants GPIO_ACTIVE_LOW and GPIO_ACTIVE_HIGH afterwards instead of 1 and 0 in the gpio parameters.

Use the Key constants instead of the hex values in linux,code.

Have a look at https://git.lede-project.org/eba84bee4ce518dc509fe51f252c5307dccf3a1c for an example.